docs/site/soap-calculator-tutorial-scaffolding.md

```diff
@@ -12,12 +12,14 @@ Let's start by creating the initial application by running the following
 command:
 
 ```sh
-lb4 app soap-calculator --enableRepository
+lb4 app soap-calculator --enableRepository --enableServices
```

Comment:
Hmm, now we have `enableRepository` vs `enableServices`. The naming is inconsistent.